src/main/java/mate/academy/bookstore/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.bookstore.service.impl;
+
+import java.util.List;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.repository.BookRepository;
+import mate.academy.bookstore.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    @Autowired
+    private BookRepository bookRepository;
+
+    @Override
+    public Book save(Book book) {
+        return bookRepository.save(book);
```

Comment:
You need to implement methods save and findAll in BookRepositoryImpl class
Please, don't use JpaRepository